### ISSUE TITLE
False negative for `Layout/IndentationWidth` with preceeding wrongly indented comment

### ISSUE THREAD

**Issue by Earlopain** [rubocop/rubocop#13168]
## Expected behavior

```rb
 # frozen_string_literal: true

 class Test
 end
```

The code is consistently indented with one space. Remove the comment, an offense is registered.

## Actual behavior

No offense

## RuboCop version

```
$ [bundle exec] rubocop -V
1.65.1 (using Parser 3.3.1.0, rubocop-ast 1.32.1, running on ruby 3.2.2) [x86_64-linux]
  - rubocop-md 1.2.2
  - rubocop-minitest 0.34.3
  - rubocop-packaging 0.5.2
  - rubocop-performance 1.20.1
  - rubocop-rails 2.23.1
```

**Comment by koic** [comment]
It seems to be the role of `Layout/InitialIndentation` rather than `Layout/IndentationWidth`. Wouldn't that work properly?

**Comment by Earlopain** [comment]
Yup, you're right. Codebase uses `DisabledByDefault: true` so I didn't realize this is handled by a different cop.
